packages/pn-pa-webapp/cypress/fixtures/recipients/recipients.ts

```diff
@@ -0,0 +1,23 @@
+export const cesare = {
```

Comment:
Since you chose to go with this solution then update all the other tests to make them using this fixture file. Move the recipients.ts file one level up and remove the recipients folder with the json files which will be no longer used.